packages/contracts/src/networks.ts

```diff
@@ -103,8 +103,8 @@ export const SPHINX_NETWORKS: Array<SupportedNetwork> = [
     name: 'ethereum',
     displayName: 'Ethereum',
     chainId: BigInt(1),
-    rpcUrl: () =>
-      `https://eth-mainnet.g.alchemy.com/v2/${process.env.ALCHEMY_API_KEY}`,
+    rpcUrl: () => process.env.ETH_MAINNET_URL!,
+    rpcUrlId: 'ETH_MAINNET_URL',
```

Comment:
Switching to using a separate env variable for every RPC URL adds a bit of repetition to our .env files but makes the integration with our secret manager much simpler. I think this is a fine tradeoff since we can easily sync our RPC URLs to the mono repo using `yarn sync:rpc`.